tests/workbenches/conftest.py

```diff
@@ -62,6 +76,17 @@ def default_notebook(
     # Set the correct username
     username = get_username(dyn_client=admin_client)
 
+    # Set the image path based on internal image registry status
+    try:
+        internal_image_registry.wait_for_condition(
+            condition=ClusterOperator.Condition.AVAILABLE,
+            status=ClusterOperator.Condition.Status.TRUE,
+            timeout=Timeout.TIMEOUT_30SEC,
```

Comment:
I don't insist, but just checking - isn't this too big timeout? :thinking:

Reply:
It's somewhat better now when there is separate check for initial existence.

There is no predefined shorter timeout constant. Better be safe and wait 30s I thought